packages/frontend/src/home.css

```diff
@@ -26,19 +26,57 @@
   }
 }
 
+.landing {
+  display: flex;
+  margin: 50px;
+}
+
+.join {
+  margin: auto;
+  text-align: center;
+  font-size: 1.3em;
+}
+
+.icon {
+  margin: auto 20px;
+  cursor: pointer;
+}
+
+.icon-sm {
+  margin-right: 2px;
+  cursor: pointer;
+}
+
 .title1 {
   font-size: 18px;
   font-weight: 600;
+  margin-bottom: 3px;
+}
+
+.title2 {
+  display: flex;
+  font-size: 18px;
+  font-weight: 600;
+  margin-bottom: 3px;
+  justify-content: center;
 }
 
 .info-container {
-  border: 1px solid black;
-  padding: 2px;
-  margin: 10px 0px;
+  display: flex;
+  border: 3px solid yellowgreen;
```

Comment:
I think the `yellowgreen` color is okay, but can we keep the outlines black? I'm imagining just a bit of color to apply a slight tint.

Reply:
Sure. I'm trying to catch up with the vision here, not hijack it ; )

Reply:
no worries, it's a collaborative effort